### DISCUSSION TITLE
Any plans to release a new version?

### DISCUSSION THREAD

**Discussion by maltoze** [pmndrs/zustand#269] [General] [answered]
I can't wait!

**Comment by dai-shi** [comment, marked as answer]
We have some changes in persist middleware that is likely to be reverted (improved), so I'm hesitant to release the current master branch. Cc @AnatoleLucet

**Reply by AnatoleLucet** [reply]
I'm still struggling with #267 and don't have much free time to work on it... I'll open a pr today for the rest (#248, #243, and some more changes here and there).
